content/en/docs/Writing policies/generate.md

```diff
@@ -22,6 +22,10 @@ Deleting the policy containing a `generate` rule with a `data` object and `synch
 
 Kubernetes has many default resource types even before considering CustomResources defined in CustomResourceDefinitions (CRDs). While Kyverno can generate these CustomResources as well, both these as well as certain default Kubernetes resources may require granting additional privileges to the ClusterRole responsible for the `generate` behavior. To enable Kyverno to generate these other types, edit the ClusterRole typically named `kyverno:generatecontroller` and add or update the rules to cover the resources and verbs needed.
 
+{{% alert title="Note" color="info" %}}
+For generating a custom resource using generate policy, it is necessary to mention the resource `apiVersion` and `kind`.
```

Comment:
Maybe better to use `set`?
...necessary to ~mention~ **set** the...

Can we be more specific on these two tags `apiVersion` and `kind`? Is it `cpol.generate.apiVersion`?